.github/Dockerfile

```diff
@@ -0,0 +1,55 @@
+ARG PHP_VERSION
+FROM php:${PHP_VERSION}-apache
+LABEL maintainer="Adyen <magento@adyen.com>"
+
+ENV MAGENTO_HOST="<will be defined>" \
+DB_SERVER="<will be defined>" \
+DB_PORT=3306 \
+DB_NAME=magento \
+DB_USER=magento \
+DB_PASSWORD=magento \
+DB_PREFIX=m2_ \
+ELASTICSEARCH_SERVER="<will be defined>" \
+ELASTICSEARCH_PORT=9200 \
+ELASTICSEARCH_INDEX_PREFIX=magento2 \
+ELASTICSEARCH_TIMEOUT=15 \
+ADMIN_NAME=admin \
+ADMIN_LASTNAME=admin \
+ADMIN_EMAIL=admin@example.com \
+ADMIN_USERNAME=admin \
+ADMIN_PASSWORD=admin123 \
```

Comment:
Should we setup these instances with stronger credentials and move them to Github secrets? They are ephemeral but good practice to do so

Reply:
These are default values we must set when declaring environment variables. They are actually overwritten by secrets at runtime: https://github.com/Adyen/adyen-magento2/blob/9890f0fa1060e288ece13d0097e43106ca85df66/.github/workflows/integration-test.yml#L18.